Clang on macOS fails on xml.c if free(3) and realloc(3) are not
declared.  As of 2.12 libxml2 does not appear to include stdlib.h.
--- a/src/xml.c
+++ b/src/xml.c
@@ -22,6 +22,8 @@
 #    include "config.h"
 #endif
 
+#include <stdlib.h>
+
 #ifdef HAVE_ERR_H
 #    include <err.h>
 #endif
